internal/lib/stats/cpu_metrics.go

```diff
@@ -0,0 +1,113 @@
+package statsserver
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cri-o/cri-o/internal/config/cgmgr"
+	"github.com/cri-o/cri-o/internal/lib/sandbox"
+	"github.com/cri-o/cri-o/internal/oci"
+	"github.com/sirupsen/logrus"
+	types "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func GenerateSandboxCPUMetrics(sb *sandbox.Sandbox, c *oci.Container, stats interface{}) []*types.Metric {
+	cpu, ok := stats.(*cgmgr.CPUStats)
+	if !ok {
+		logrus.Errorf("Failed to assert stats as *cgmgr.CpuStats")
+		return nil
+	}
+
+	cpuMetrics := []*containerMetric{
+		{
+			desc: &types.MetricDescriptor{
+				Name:      "container_cpu_user_seconds_total",
+				Help:      "Cumulative user CPU time consumed in seconds.",
```

Comment:
The names and corresponding help messages should be const values and reused in `PopulateMetricDescriptors`.